charts/azure-aks-aso/README.md

```diff
@@ -58,6 +62,24 @@ managedMachinePoolSpecs:
 helm install <name> capi/azure-aks-aso -f values.yaml
 ```
 
+## Install the CAPZ AKS-ASO Helm chart with ClusterClass
+
+Be sure to set both of these values to `true` in the values.yaml file.
+```yaml
+withClusterClass: true
+withClusterTopology: true
+```
+
+```
+helm install <name> capi/azure-aks-aso -f values.yaml
+```
+
+On the first install, there will be this message and it is expected since the clusterclass needs to provision first.  It will eventually reconcile successfully.
```

Comment:
Can you specify that both values should only be set to `true` once to create the class, and any subsequent clusters that are created with that ClusterClass will need to set `withClusterClass` to `false`?

Maybe renaming `withClusterClass` or introducing a new value `clusterClassName` would make this flow clearer. Could be something for the future though.